### PR TITLE
Use pre-built ubuntu nsenter image in CI

### PR DESCRIPTION
**Pull request by munnerz**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

/assign @wallrj

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,8 +14,7 @@ jobs:
       - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
       - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.21.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
       # Install nsenter
-      - docker build -t ubuntu-nsenter ./hack/ubuntu-nsenter
-      - docker run -v /usr/local/bin:/hostbin ubuntu-nsenter cp /nsenter /hostbin/nsenter
+      - docker run -v /usr/local/bin:/hostbin munnerz/ubuntu-nsenter cp /nsenter /hostbin/nsenter
```
Comment (wallrj) [inline .travis.yml]: * This should be a Jetstack image.
* Maybe make it a Dockerhub automated build? https://docs.docker.com/docker-hub/builds/
* Or at least add a note to the dockerhub page explaining where the Dockerfile is.

**Comment by munnerz** [comment]
@wallrj done - waiting for image to build, which might cause tests to fail first time https://github.com/jetstack/ubuntu-nsenter

File: .travis.yml [hunk 2]
```diff
@@ -14,8 +14,7 @@ jobs:
       - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
       - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.21.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
       # Install nsenter
-      - docker build -t ubuntu-nsenter ./hack/ubuntu-nsenter
```
Comment (wallrj) [inline .travis.yml]: We should remove this directory if we no longer use it.

**Pull request merged**